analytics_data_api/v0/views/courses.py

```diff
@@ -32,7 +32,7 @@ def get(self, request, *args, **kwargs):
         self.course_id = self.kwargs.get('course_id')
         start_date = request.query_params.get('start_date')
         end_date = request.query_params.get('end_date')
-        timezone = utc
+        timezone = datetime.timezone.utc
```

Comment:
#### Reason for this change

> The [django.utils.timezone.utc](https://docs.djangoproject.com/en/4.2/ref/utils/#django.utils.timezone.utc) alias to [datetime.timezone.utc](https://docs.python.org/3/library/datetime.html#datetime.timezone.utc) is deprecated. Use [datetime.timezone.utc](https://docs.python.org/3/library/datetime.html#datetime.timezone.utc) directly.

https://github.com/edx/video-encode-manager/blob/6cf71026b8c5964d980dcd57abf41a46ee1976a6/Makefile#L106